manifests/cluster-monitoring-operator-role.yaml

```diff
@@ -10,9 +10,7 @@
 # 	assets/node-exporter/cluster-role.yaml
 # 	assets/prometheus-k8s/cluster-role.yaml
 # 	assets/prometheus-k8s/role-config.yaml
-# 	assets/prometheus-k8s/role-default.yaml
-# 	assets/prometheus-k8s/role-kube-system.yaml
-# 	assets/prometheus-k8s/role-namespace.yaml
+# 	assets/prometheus-k8s/role-specific-namespaces.yaml
```

Comment:
This file is a list of roles, so we need special logic in `hack/merge_cluster_roles.py` to treat it correctly.

Reply:
We also need to handle creating the list of roles and rolebindings now. Previously we handled them individually.